src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueTest.java

```diff
@@ -60,7 +60,10 @@ void whenMeshInboundQueueMessageIsReceivedThenMessageIsHandled() throws IOExcept
                 new Customization("identifier.value", IGNORE),
                 new Customization("entry[*].fullUrl", IGNORE),
                 new Customization("entry[*].resource.**.reference", IGNORE),
-                new Customization("entry[*].resource.id", IGNORE)
+                new Customization("entry[*].resource.id", IGNORE),
+                new Customization("entry[*].resource.subject.reference", IGNORE),
+                new Customization("entry[*].resource.id", IGNORE),
+                new Customization("entry[*].resource.performer[*].actor.reference", IGNORE)
```

Comment:
This should be updated to match the other JSON Comparators

Reply:
Updated in 3324155 to match other JSON Comparators